app/views/shopping_list/index.html.erb

```diff
@@ -0,0 +1,42 @@
+ <div class="container pb-2">
+    <h2 class="text-center">
+     The Shopping List
+    </h2>
+
+    <div class="d-flex flex-row ">
+       <div class="flex-grow-1">
+           The amount of food items to buy: <%= @ingredients.length %>
+       </div>
+       <div class="flex-grow-1">
+           The total value of food needed: $<%= @total_price %>
+       </div>
+    </div>
+ </div>
+
+ <div class="container">
+    <table class="table table-bordered table-striped" >
+      <thead>
+        <tr class="text-center">
+          <th>Food</th>
+          <th>Quantity</th>
+          <th>Price</th>
+        </tr>
+      </thead>
+      <tbody class="text-center">
+        <% @ingredients.each do |food, ingredients| %>
+        <tr>
+          <td><%= food %></td>
+          <td>
+              <% total_quantity = ingredients.map(&:quantity).sum %>
+              <span><%= total_quantity %> <%= ingredients.first.food.measurement_unit %></span>
+          </td>
+          <td>
+              <% total_price = ingredients.map { |ingredient| ingredient.quantity * ingredient.food.price }.sum %>
+              $<%= total_price %>
+          </td>
+        </tr>
+      <% end %>
+      </tbody>
+    </table>
```

Comment:
- [ ] **[OPTIONAL]** Also, I have a take on the shopping list: I'm sorry that the previous reviewers didn't point this out so I'm marking this as optional to keep consistency between reviews, but according to the requirements:

![image](https://user-images.githubusercontent.com/77069011/233796831-31c03ff2-3159-4e08-812d-f1eb3d3ec9f1.png)

The goal of this general shopping list is to list the missing food for all the recipes of the user, so for example, if the user has 5kg of `food1`, and one recipe asks for 10kg, the shopping list should show 5kg and not 10kg, which is what the user needs to buy in order to have enough for the recipe, take in mind that it should list also ingredients from other recipes if there's more than one! 🤓 🔧  

💡 Hint: Compare the `recipe_foods` quantity for the user's recipes with the user's `foods` quantity.

Currently, your app only takes the `recipe_foods` quantity and prints this in the shopping list, without taking into account the amount of food the user already has! I strongly suggest that you fix this! 🤓 🔧